db/patch68.sql

```diff
@@ -3,4 +3,4 @@
 -- This resolves https://joindin.jira.com/browse/JOINDIN-725
 ALTER DATABASE joindin CHARACTER SET utf8 COLLATE utf8_unicode_ci;
 
-INSERT INTO patch_history SET patch_number = 68;
\ No newline at end of file
+INSERT INTO patch_history SET patch_number = 69;
```

Comment:
I would disagree there @exussum12, the person accepting the the merge should not be making any changes at all to the code, this should be done by the person that creates the change in the first place.